### PR TITLE
Update linux-resources.md

### PR DESCRIPTION
**Pull request by msbemba**
removed unused from EDR as it is now under general availability

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9785

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/linux-resources.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ The following table lists commands for some of the most common scenarios. Run `m
 |Quarantine management|Add a file detected as a threat to the quarantine|`mdatp threat quarantine add --id [threat-id]`|
 |Quarantine management|Remove a file detected as a threat from the quarantine|`mdatp threat quarantine remove --id [threat-id]`|
 |Quarantine management|Restore a file from the quarantine|`mdatp threat quarantine restore --id [threat-id] --path [destination-folder]`|
-|Endpoint Detection and Response|Set early preview (unused)|`mdatp edr early-preview [enable|disable]`|
+|Endpoint Detection and Response|Set early preview |`mdatp edr early-preview [enable|disable]`|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-resources.md]:
```suggestion
|Endpoint Detection and Response|Set early preview |`mdatp edr early-preview [enabled\|disabled]`|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, not related but could you also commit the additional change, please?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder

**Pull request merged**